### DISCUSSION TITLE
Problem synthesizing simple words

### DISCUSSION THREAD

**Discussion by mathias-ewald** [coqui-ai/TTS#1088] [General Q&A]
I am not sure how to even describe the result I am getting when I want a simple "hello" to be pronounced. The result is a bit like "helooOOooOoo OOoooOOooosss" Hard to explain :D Please check the attached sample.

[example.zip](https://github.com/coqui-ai/TTS/files/7834886/example.zip)

**Comment by erogol** [comment]
What model? What command did you run?

**Comment by mathias-ewald** [comment]
I used the docker image from https://github.com/synesthesiam/coqui-docker and ran the tts-server command. The output I am getting starting the container is

```
> tts_models/en/ljspeech/tacotron2-DDC is already downloaded.
 > vocoder_models/en/ljspeech/hifigan_v2 is already downloaded.
 > Using model: Tacotron2
 > Vocoder Model: hifigan
 > Generator Model: hifigan_generator
 > Discriminator Model: hifigan_discriminator
Removing weight norm...
 * Serving Flask app 'TTS.server.server' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://[::1]:5002/ (Press CTRL+C to quit)
```

I hope that answers your question.

**Comment by erogol** [comment]
You can try out the model here.  https://huggingface.co/spaces/erogol/CoquiTTS

You need to put punctuation to the end. Without punctuation, model does not know where to stop.
